source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
+| *8.0.0*     |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | 2018-09-06   | Late 2019      | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.0.0       | 2018-01-01   | ^              | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 6.0.0       | 2017-05-02   | 2017-12-31     | * SpongeForge (1.11.2)                    |
+|             |              |                | * SpongeVanilla (1.11.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 5.1.0       | 2017-02-25   | 2017-12-31     | * SpongeForge (1.10.2)                    |
+|             |              |                | * SpongeVanilla (1.10.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+
+It is recommended to always update to the latest stable version of the API or to an LTS version.
+
+.. note::
+
+    You can still release your plugin with an older major API version, but we don't officially support these versions.
+
+
+LTS-Versions
+============
+
+The following versions are LTS (long time support) versions.
+
+* SpongeAPI 7.x until late 2019
+
+
+Getting the API Version from Implementations
+============================================
+
+Getting it from the Jar
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* For `SpongeForge <https://www.spongepowered.org/downloads/spongeforge/>`__: 
+
+  **Example:** ``spongeforge-1.12.2-2705-7.1.0-BETA-3361.jar``
+  
+  => API-Version ``7.1.0`` (might reference a preview though)
```

Comment:
I'm not sure what "might reference a preview though" means in this context. This might need clarification or elucidation. If the meaning of this is obvious to users, then ignore me.

Reply:
Currently the version numbering does not clarify whether the mentioned version (7.1.0) has been released already. As you can see on the downloads page, the reference points to 7.1.0 for quite some time already, although 7.1.0 has just been released recently. Aka it targets a preview, pre-release or snapshot.

Maybe `may refer to a SNAPSHOT` would be less elusive?

Reply:
The idea that needs communicating is:

The version string in SpongeForge & SpongeVanilla is currently flawed, it currently has a minor version number referring to **unreleased** API versions, and is planned to be fixed in an upcoming update.

As we follow SemVer, changes in 7.1.0 should be backwards compatible with 7.0.0, as such, we often ship SpongeForge & SpongeVanilla with **preview** implementations of the upcoming API release.

The API contained within the preview API builds can and will change before it's released, so for plugin compatibility sake, it's best to use plugins for the latest stable released API, and not the API preview versions.

As such, if the API version number in the filename points to 7.2.0 but 7.2.0 is not yet announced to be released, you should use plugins built against 7.1.0 unless specifically testing new functionality as asked by plugin devs, or you are willing to put up with small breakages occasionally for the sake of new features.

This **WILL** be fixed in the near future. When this happens the Version number of SpongeForge & SpongeVanilla will no longer reference unreleased SpongeAPI minor versions, but will still ship previews of the next minor API for testing purposes.

Major API version bumps will be reserved for the experimental / bleeding builds, and will be very explicit as their major version number will be different.

Reply:
@ryantheleach See my comment on https://github.com/SpongePowered/SpongeForge/issues/2367#issuecomment-420204233

Reply:
I copied the block into the docs because it pretty much explains the issue.